crates/sp-lightclient/src/lib.rs

```diff
@@ -128,6 +130,8 @@ pub trait HeaderImporter<Header: HeaderT> {
         let parent_header = Self::Storage::header(BlockNumberOrHash::Hash(parent_hash))
             .ok_or_else(|| ImportError::MissingParent(header.hash()))?;
 
+        // TODO(ved): check for farmer equivocation
```

Comment:
I thought we agreed to not do that in light client, didn't we?

Reply:
Dont think so. We both agreed that its not in the header at the moment but evetually could be moved to header from unsigned. 
We dont need to fix the TODO at the moment though :wink: